rfc3164/example_test.go

```diff
@@ -78,6 +78,28 @@ func Example_withtimezone() {
 	// })
 }
 
+func Example_withlocaletimezone() {
+	pst, _ := time.LoadLocation("PST")
```

Comment:
please use `time.LoadLocation("America/New_York")` instead. `time.LoadLocation("PST")` will not work.